test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -91,5 +91,26 @@ public EndToEndTests(ServerFixture serverFixture)
 
             return null;
         }
+
+        [Fact]
+        public async Task ConnectionCanSendAndReceiveMessagesWebSocketsTransport()
```

Comment:
We need transport specific tests. Look at what we have in LongpollingTransportTests.